src/index.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import ListContainer from './ListContainer';
+import ListItemComponent from './ListItemComponent';
+
+const redditAPI = 'https://www.reddit.com/r/';
+
+export default class SubredditPosts extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isLoading: true,
+      redditPosts: []
+    };
+  }
+
+  componentDidMount() {
+    const uri = `${redditAPI}${this.props.subreddit}.json`;
+
+    fetch(uri)
+      .then(data => data.json())
+      .then(this.handlePosts)
+      .catch(err => console.error(err));
+  }
+
+  handlePosts = (posts) => {
```

Comment:
Oh yeah, the fat arrow functions automatically bind `this`, so you no longer need to define the binds in the constructor.

Reply:
so you changed it from an arrow function and back to: 

Oh nice, would it be recommended to use arrow functions in components to avoid that extra step and that extra LOC?

Reply:
Yeah, fat arrow functions are the recommended way of defining functions which require the use of `this`. However it's not worth putting them on lifecycle methods as it doesn't make a difference and can hurt performance.